File: Java Basics/src/Conditions/Scolarship.java
package Conditions;

import java.util.Scanner;

public class Scolarship {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        double income = Double.parseDouble(scan.nextLine());
        double averageGrade = Double.parseDouble(scan.nextLine());
        double minWage = Double.parseDouble(scan.nextLine());

        double socialScolarshipSize = minWage * 35 / 100.0;
        double excellentScolarshipSize = averageGrade * 25;

        boolean socialScolarship = income <= minWage && averageGrade >= 4.5;
        boolean excelentScolarship = averageGrade >= 5.5;

        if ((socialScolarship && !excelentScolarship) ||
            (socialScolarship && socialScolarshipSize > excellentScolarshipSize)) {
            System.out.printf("You get a Social scholarship %.0f BGN", socialScolarshipSize);
        } else if ((!socialScolarship && excelentScolarship) ||
                   (socialScolarship && socialScolarshipSize <= excellentScolarshipSize)) {
            System.out.printf("You get a scholarship for excellent results %.0f BGN", excellentScolarshipSize);
        } else {
            System.out.println("You cannot get a scholarship!");
        }
    }
}

File: C# Basics/Conditions/Scolarship.cs
using System;
using System.Collections.Specialized;
using System.Net.Sockets;

namespace Scolarship
{
    class Program
    {
        static void Main(string[] args)
        {
            double income = double.Parse(Console.ReadLine());
            double grade = double.Parse(Console.ReadLine());
            double minWage = double.Parse(Console.ReadLine());

            double socialScholarship = Math.Floor(minWage * (35.0 / 100));
            double gradeScolarship = Math.Floor(grade * 25);

            string highGrade = $"You get a scholarship for excellent results {gradeScolarship} BGN";
            string social = $"You get a Social scholarship {socialScholarship} BGN";

            if (income <= minWage && grade >= 5.50 && gradeScolarship >= socialScholarship)
            {
                Console.WriteLine(highGrade);
            }
            else if (income <= minWage && grade >= 5.5 && gradeScolarship < socialScholarship)
            {
                Console.WriteLine(social);
            }
            else if (income > minWage && grade >= 5.5)
            {
                Console.WriteLine(highGrade);
            }
            else if (income <= minWage && grade > 4.5)
            {
                Console.WriteLine(social);
            }
            else
            {
                Console.WriteLine("You cannot get a scholarship!");
            }

        }
    }
}
